backend/api/v1/routers/change_beneficiary_orientation.py

```diff
@@ -0,0 +1,81 @@
+import logging
+import os
+from uuid import UUID
+
+from fastapi import APIRouter, Depends, Header
+from fastapi.exceptions import HTTPException
+from gql import Client, gql
+from gql.transport.aiohttp import AIOHTTPTransport
+from pydantic import BaseModel
+
+from api.core.settings import settings
+from api.db.models.role import RoleEnum
+from api.v1.dependencies import allowed_jwt_roles
+
+orientation_manager_only = allowed_jwt_roles([RoleEnum.ORIENTATION_MANAGER])
+router = APIRouter(dependencies=[Depends(orientation_manager_only)])
+
+logger = logging.getLogger(__name__)
+
+
+class ChangeBeneficiaryOrientationInput(BaseModel):
+    orientation_request_id: UUID | None
+    orientation_type: str
+    notebook_id: UUID
+    beneficiary_id: UUID
+    structure_id: UUID
+    professional_account_id: UUID | None
+
+    def gql_variables(self):
```

Comment:
```suggestion
    def gql_variables(self) -> dict[str, str | bool]:
```
Si j'ai bien compris le type ?